scripts/library.sh

```diff
@@ -528,10 +528,19 @@ function run_go_tool() {
 # Run kntest tool, error out and ask users to install it if it's not currently installed.
 # Parameters: $1..$n - parameters passed to the tool.
 function run_kntest() {
-  if [[ -z "$(which kntest)" ]]; then
```

Comment:
Can we do this in a separate PR? As well as changes in other shell scripts

Reply:
Sure, I have reverted the script changes in this PR, and will create another PR for it.

Reply:
The e2e tests cannot pass without the changes in `test/e2e-tests.sh`, so I'm keeping that file and will create another PR to only change `scripts/library.sh`